module/VuFind/src/VuFind/Recommend/AbstractSearchObject.php

```diff
@@ -61,6 +61,20 @@ abstract class AbstractSearchObject implements RecommendInterface
      */
     protected $limit;
 
+    /**
+     * Heading for this recommendation module
+     *
+     * @var string
+     */
+    protected $heading;
+
+    /**
+     * Config section with filters for this search
+     *
+     * @var string
+     */
+    protected $iniSection;
```

Comment:
Maybe call this $filterIniSection in case we have another .ini section for some other thing in future?

Reply:
Good call